### PR TITLE
Support new os C-DATA

### PR DESCRIPTION
**Pull request by hrtrd**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by hrtrd** [comment]
@PipoCanaja its possible ignore codeclimate for that 3 issues? Or how to edit this? (best practice)

**Comment by hrtrd** [comment]
i think "Travic CI" stuck

File: mibs/cdata/SNMPv2-MIB [hunk 1]
```diff
@@ -0,0 +1,773 @@
+-- Changes to rfc1907 - SNMPv2-MIB (object for SNMPv2 protocol)
+--      NOTE: the following should be in a group -
```
Comment (PipoCanaja) [inline mibs/cdata/SNMPv2-MIB]: Please remove any standard mib file (that is already in ./mibs/) from this directory. Only vendor specific mibs should be here.
If a standard mib is missing from ./mibs/, then please add it there instead od the vendor specific directory

File: mibs/cdata/SNMPv2-SMI [hunk 2]
```diff
@@ -0,0 +1,119 @@
+SNMPv2-SMI DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/cdata/SNMPv2-SMI]: Please remove any standard mib file (that is already in ./mibs/) from this directory. Only vendor specific mibs should be here.
If a standard mib is missing from ./mibs/, then please add it there instead od the vendor specific directory

**Comment by PipoCanaja** [comment]
You can ignore CodeClimate. Travis will be run again anyway after the move of the mib files.

**Comment by hrtrd** [comment]
> A few mib files to remove.

Okey, deleted

File: mibs/cdata/BRIDGE-MIB [hunk 3]
```diff
@@ -0,0 +1,1478 @@
+-- *****************************************************************
```
Comment (PipoCanaja) [inline mibs/cdata/BRIDGE-MIB]: This is also a standard one :)

Comment (hrtrd) [reply mibs/cdata/BRIDGE-MIB]: done )

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-59-release-changelog-december-2019/10558/1

**Pull request merged**